### ISSUE TITLE
Word space issues across the 14 opsz variable design plane

### ISSUE THREAD

**Issue by EbenSorkin** [TypeNetwork/Roboto-Flex#164]
It seems like the word space at a specific opsz (14 or 8 or 144) should remain the same across variations in slnt, GRAD, and wght. I am not seeing this and I think it should be fixed.

I will admit that tradition could make Wght and word space matching seem odd. However since Roboto flex is meant to be adapting to opsz in other ways and the 900 wght is meant to be just as text-oriented as 400 @ 14 opsz, I think it does make sense for them to match. 

I want to add however that I don't think is true when thinking about width. The word space probably does have to change for that variable because it has to adapt to new counter sizes. But again across the variables above and at 25, 100, and 150(151?) width I again do think that we should see word spaces matching.

100 wdth: ( default ) here the light wght: 100 | wdth: 100 | slnt: -10 seems to tight 

The heavy wght: 900 | wdth: 100 | in both slnt: -10  and slnt: 0 seems to be too wide.

25 wdth:  the word space seems to be in fairly decent shape in general except for the wght: 100 | wdth: 150 | slnt: -10  which seems to need to have a wider word space.

150 wdth: There seems to be variation here but differences seem subtle except again for wght: 100 | wdth: 150 | slnt: -10 although the difference is less problematic than 25 wdth.

**Comment by dberlow** [comment]
You are correct in the first sentence. I agree to try word spaces matching over slnt, GRAD, and wght, per size. Width's space should expand and contract consistently as well over size.

The examples are not well itemized though, so it's hard to figure your preferences out of them. I think we've done this by script a number of ways but now should just get consistency over slnt, GRAD, and wght, per-size & wdth's overall range, and then widths's per-size range, unless the above list can be parsed into changes that also reflect that.

**Comment by EbenSorkin** [comment]
I have not made comments about the values because I have not checked what the values are. But if a script can be run to make all the word spaces attached to a specific opsz have the same width value - that would fix this. 

What I am not clear about and am wondering about is if there is something about the split controls for vertical contrast vs horizontal or counters vs exterior shapes or that in some way complicates or has the potential to defeat that fix.
